database/fake_data.sql

```diff
@@ -0,0 +1,295 @@
+('2024-10-01 07:06:00', '2024-10-01 19:21:00', 37),
+('2024-10-01 07:07:00', '2024-10-01 19:22:00', 38),
+('2024-10-01 07:08:00', '2024-10-01 19:23:00', 39),
+('2024-10-01 07:09:00', '2024-10-01 19:24:00', 40),
+('2024-10-01 07:10:00', '2024-10-01 19:25:00', 41),
+('2024-10-01 07:11:00', '2024-10-01 19:26:00', 42),
+('2024-10-01 07:12:00', '2024-10-01 19:27:00', 43),
+('2024-10-01 07:13:00', '2024-10-01 19:28:00', 44),
+('2024-10-01 07:14:00', '2024-10-01 19:29:00', 45),
+('2024-10-01 07:15:00', '2024-10-01 19:30:00', 46),
+('2024-10-01 07:16:00', '2024-10-01 19:31:00', 47),
+('2024-10-01 07:17:00', '2024-10-01 19:32:00', 48),
+('2024-10-01 07:18:00', '2024-10-01 19:33:00', 49),
+('2024-10-01 07:19:00', '2024-10-01 19:34:00', 50),
+('2024-10-01 07:20:00', '2024-10-01 19:35:00', 51),
+('2024-10-01 07:21:00', '2024-10-01 19:36:00', 52),
+('2024-10-01 07:22:00', '2024-10-01 19:37:00', 53),
+('2024-10-01 07:23:00', '2024-10-01 19:38:00', 54),
+('2024-10-01 07:24:00', '2024-10-01 19:39:00', 55),
+('2024-10-01 07:25:00', '2024-10-01 19:40:00', 56),
+('2024-10-01 07:26:00', '2024-10-01 19:41:00', 57),
+('2024-10-01 07:27:00', '2024-10-01 19:42:00', 58),
+('2024-10-01 07:28:00', '2024-10-01 19:43:00', 59),
+('2024-10-01 07:29:00', '2024-10-01 19:44:00', 60),
+('2024-10-01 07:30:00', '2024-10-01 19:45:00', 61),
+('2024-10-01 07:31:00', '2024-10-01 19:46:00', 62),
+('2024-10-01 07:32:00', '2024-10-01 19:47:00', 63),
+('2024-10-01 07:33:00', '2024-10-01 19:48:00', 64),
+('2024-10-01 07:34:00', '2024-10-01 19:49:00', 65),
+('2024-10-01 07:35:00', '2024-10-01 19:50:00', 66),
+('2024-10-01 07:36:00', '2024-10-01 19:51:00', 67),
+('2024-10-01 07:37:00', '2024-10-01 19:52:00', 68),
+('2024-10-01 07:38:00', '2024-10-01 19:53:00', 69),
+('2024-10-01 07:39:00', '2024-10-01 19:54:00', 70),
+('2024-10-01 07:40:00', '2024-10-01 19:55:00', 71),
+('2024-10-01 07:41:00', '2024-10-01 19:56:00', 72),
+('2024-10-01 07:42:00', '2024-10-01 19:57:00', 73),
+('2024-10-01 07:43:00', '2024-10-01 19:58:00', 74),
+('2024-10-01 07:44:00', '2024-10-01 19:59:00', 75),
+('2024-10-01 07:45:00', '2024-10-01 20:00:00', 76),
+('2024-10-01 07:46:00', '2024-10-01 20:01:00', 77),
+('2024-10-01 07:47:00', '2024-10-01 20:02:00', 78),
+('2024-10-01 07:48:00', '2024-10-01 20:03:00', 79),
+('2024-10-01 07:49:00', '2024-10-01 20:04:00', 80),
+('2024-10-01 07:50:00', '2024-10-01 20:05:00', 81),
+('2024-10-01 07:51:00', '2024-10-01 20:06:00', 82),
+('2024-10-01 07:52:00', '2024-10-01 20:07:00', 83),
+('2024-10-01 07:53:00', '2024-10-01 20:08:00', 84),
+('2024-10-01 07:54:00', '2024-10-01 20:09:00', 85),
+('2024-10-01 07:55:00', '2024-10-01 20:10:00', 86),
+('2024-10-01 07:56:00', '2024-10-01 20:11:00', 87),
+('2024-10-01 07:57:00', '2024-10-01 20:12:00', 88),
+('2024-10-01 07:58:00', '2024-10-01 20:13:00', 89),
+('2024-10-01 07:59:00', '2024-10-01 20:14:00', 90),
+('2024-10-01 08:00:00', '2024-10-01 20:15:00', 91);
+
+
+INSERT INTO body_assignment (region_id, body_id, at, azimuth, altitude)
+VALUES
```

Comment:
Maybe include negative values for this section

Reply:
Great mappings here, we can use this to test fetching astronomical positional data from the astronomyAPI for a given 'night period' (sunset). Of course, also allows us to test our database stores data as we intend.